### ISSUE TITLE
[Bug]:  bug(a11y): sbb-menu not announced as list

### ISSUE THREAD

**Issue by MarioCastigliano** [sbb-design-systems/lyne-components#4065]
### Preflight Checklist

- [x] I have read the [Contributing Guidelines](https://github.com/sbb-design-systems/lyne-components/blob/main/docs/CONTRIBUTING.md) for this project.
- [x] I agree to follow the [Code of Conduct](https://github.com/sbb-design-systems/lyne-components/blob/main/docs/CODE_OF_CONDUCT.md) that this project adheres to.
- [x] I have searched the [issue tracker](https://github.com/orgs/sbb-design-systems/projects/4) for an issue that matches the one I want to file, without success.

### Bug type

Accessibility

### It affects the following packages

Components

### Input mode

Mouse/Keyboard

### What happened?

When inspecting [sbb-menu List](https://lyne-storybook.app.sbb.ch/?path=/story/elements-sbb-menu-sbb-menu--list) with NVDA list elements are not announced as list (e.g element 1 of n).

Does not apply if dividers or custom content are slotted in the menu.

### Steps to reproduce the issue

1. Open menu
2. focus first element

### Relevant log output

```shell

```
